### PR TITLE
Adding overview and setup details to readme

### PR DESCRIPTION
**Pull request by nburkley**
ping @edenspiekermann/backend @eschaefer

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,73 @@
...
+
+5) Once the article is confirmed, the application will track the article using the Google Analytics Realtime API
+
+6) The slackbot will send the user a message if the analytics show a large increase or decrease in article traffic.
+
+## Development Setup
+
+### Prerequisites
+
+The app is built using Phoenix 1.2 and Elixir 1.3 and uses PostgreSQL 9.4
```
Comment (zfhui) [inline README.md]: I'd change this into a bullet point list. I think it'd give a better overview.

**Comment by zfhui** [comment]
- a TOC would be nice, but the read me is still pretty short
- I'm missing some info about the deployment and release flow, but maybe it's not relevant right now?

**Review by zfhui — APPROVED** [review]

**Comment by nburkley** [comment]
yeah, we don't have any deployment yet :) I'm going to set it up on heroku today hopefully.

File: README.md (hunk 1 above)
Comment (nburkley) [reply README.md]: yup, good point

**Comment by eschaefer** [comment]
Thanks! Added my own notes in https://github.com/edenspiekermann/espi-dni/pull/9

**Pull request merged**